.github/workflows/publish.yml

```diff
@@ -80,21 +80,16 @@ jobs:
       - name: Set up JDK 1.11
         uses: actions/setup-java@v3
         with:
-          distribution: 'temurin'
-          java-version: '11'
-
+          distribution: "temurin"
+          java-version: "11"
```

Comment:
I thought Ghidra 11.2 needs JDK 21 now? I'm surprised this still works

Reply:
Updated. Let's see how this works now.